apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -5739,6 +5747,8 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
     public checkIfNeedToExcludeTax(data: any): void {
         this.excludeTax = false;
         let isPartyTypeSez = false;
+        this.tcsTdsTaxesAccount = [];
+        let TdsTcsTaxType = ['tdsrc', 'tdspay', 'tcspay', 'tcsrc'];
```

Comment:
Avoid using PascalCase in variable names

Reply:
Done